src/client/l10n/localePathPattern.ts

```diff
@@ -0,0 +1,27 @@
+export const LOCALE_URL_PARAMS = [
+  'se',
+  'se-en',
+  'no',
+  'no-en',
+  'dk',
+  'dk-en',
+] as const
+
+export type LocaleUrlParams = typeof LOCALE_URL_PARAMS
+
+export const getLocalePathPattern = (
+  permittedLocaleUrlParams: LocaleUrlParams,
+) => {
+  const labelsString = permittedLocaleUrlParams.reduce(
+    (accumulated, currentLabel) => {
+      return accumulated.length
+        ? `${accumulated}|${currentLabel}`
+        : currentLabel
+    },
+    '',
+  )
+
+  return `/:locale(${labelsString})`
```

Comment:
Could this be simplified with an array join?
```suggestion
  return `/:locale(${permittedLocaleUrlParams.join('|')})`
```

Reply:
Haha YES! 🧠 🤦‍♀️

Reply:
(It's rarely a good sign when you feel like you've done something super smart)